swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, category="Clothing", condition=0):
+        self.category = category
+        self.condition = condition 
```

Comment:
This works, but doesn't fully take advantage of inheritance here. Lines 6-7 are exactly the same as lines 5-6 in `Item`, so we've duplicated some code. It would be preferable to use a call to `__super__` here so we can take advantage of the `Item` constructor. It doesn't do that much for us right now - we only avoid duplicating two lines of code. But it ensures that if we ever add any extra functionality to the `Item` constructor in the future we don't need to copy it down to all the child classes - they'll just get it automatically through their use of `__super__`